torch_xla/csrc/aten_cpu_fallback.cpp

```diff
@@ -51,8 +52,33 @@ std::vector<std::string> GetFallbackOperations() {
 // Before each modified function below, we shall specify what has changed,
 // if there was any.
 
-bool UseCUDAFallback() {
-  return runtime::sys_util::GetEnvBool("XLA_FALLBACK_CUDA", false);
+// Decide whether to run OpenXLA fallback operations on CUDA.
+bool UseCUDAFallback(const c10::OperatorHandle& op) {
```

Comment:
Since we have another type of cuda fallback, how about the name "OpenXLAFallbackOnCUDA"